emission/net/api/config.py

```diff
@@ -0,0 +1,48 @@
+import json
+import logging
+import os
+
+def get_config_data_from_env():
+    config_data_env = {
+        "static_path": os.getenv('WEB_SERVER_STATIC_PATH', "webapp/www/"),
+        "server_host": os.getenv('WEB_SERVER_HOST', "0.0.0.0"),
+        "server_port": os.getenv('WEB_SERVER_PORT', "8080"),
+        "socket_timeout": os.getenv('WEB_SERVER_TIMEOUT', "3600"),
+        "auth_method": os.getenv('WEB_SERVER_AUTH', "skip"),
+        "aggregate_call_auth": os.getenv('WEB_SERVER_AGGREGATE_CALL_AUTH', "no_auth"),
+        "not_found_redirect": os.getenv('WEB_SERVER_REDIRECT_URL', "https://www.nrel.gov/transportation/openpath.html")
+    }
+    return config_data_env
+
+def check_unset_env_vars():
```

Comment:
this is not actually used anywhere. Line 36 is commented out.

Reply:
Removed after testing without it